tests/ui/specs/Item/Item-specs.js

```diff
@@ -112,7 +112,7 @@ describe('Item', function () {
 
 				it('should focus an inline item with 5-way Left', function () {
 					item7Inline.focus();
-					Page.spotlightLeft();
+					Page.spotlightDown();
```

Comment:
'should focus an inline item with 5-way Left'

Page.spotlightDown()

I think the previous(spotlightLeft) is right. isn't it?